docs/getting-started/welcome.md

```diff
@@ -1,36 +1,55 @@
----
-title: Welcome to UMA
-sidebar_label: Welcome to UMA
-slug: /
----
-
-UMA is open-source infrastructure for deploying and enforcing synthetic assets on Ethereum. UMA enables developers to quickly and easily build synthetic tokens that track the price of anything! View [here](synthetic-tokens/what-are-synthetic-assets.md) to learn more about synthetic tokens!
-
-
-## What infrastructure does UMA provide?
-
-UMA enables developers to create synthetic assets via two core components:
-
-- [Priceless financial contract templates](https://docs.umaproject.org/getting-started/synthetic-tokens)
-- [Optimistic oracle service](https://docs.umaproject.org/getting-started/oracle)
-
-Together, these two components enable the creation of fast, efficient, and secure synthetic derivatives on the Ethereum blockchain.
-
-## Understanding UMA
-
-If you would like to learn more about the concepts behind UMA's priceless financial contracts and optimistic oracle, start by reading this [overview](getting-started/overview.md) of the UMA ecosystem.
-
-## For Developers
-
-If you are a developer interested in building with UMA Protocol, view this [explainer](synthetic-tokens/what-are-synthetic-assets.md) of how UMA's priceless synthetic tokens work and to learn more about UMA's expiring synthetic tokens. 
-
-Additionally, the [development guide](build-walkthrough/build-process) to learn how to launch your own expiring synthetic token on UMA. 
-
-You can find additional reference materials, including a list of UMA's testnet and mainnet contracts [here](dev-ref/addresses.md).
-
-## For UMA Tokenholders
-
-If you are interested in the UMA project token, please read this [section](uma-tokenholders/uma-holders.md) to learn more about the role of the UMA project token in the DVM and the rights and responsibilities of UMA project tokenholders.
-
-## Contact 
-If you have any questions or want to connect with the UMA community, please check out our community [page](community/community-overview) or join our [Discord](https://discord.umaproject.org/).
+---
+title: Welcome to UMA
+sidebar_label: Welcome to UMA

```

Comment:
Same comment as other PRs

Reply:
Welcome file is now overwritten by PR143